agent/ZappiehostBuyer.py

```diff
@@ -113,7 +113,10 @@ def placeOrder(self):
 
             self.driver.find_element_by_css_selector("input[type='submit'][class='cartbutton green ui-button ui-widget ui-state-default ui-corner-all']").click() # Submit the form
 
-            self.driver.find_element_by_css_selector("input[type='submit'][value='Pay Now']").click()
+            try:
+                self.driver.find_element_by_css_selector('input[value="Pay Now"]').click()
+            except Exception as e:
+                print("Warning: Pay now button not found")

```

Comment:
While i agree it is not a great solution i have a hard time coming up with a better solution.
With this at least the programmer knows something is up.
Maybe we should define an error to be thrown by VPSBuyer in case a VPSBuyer failed? Then if this error is thrown birthchamber could simply pick a different vps instead.
although this wont be done before i' m feeling confident about the documentation being back up to scratch etc. the error could be defined and thrown here instead of the print statement